tests/system/aiplatform/test_dataset.py

```diff
@@ -227,186 +164,156 @@ def test_get_nonexistent_dataset(self):
         """Ensure attempting to retrieve a dataset that doesn't exist raises
         a Google API core 404 exception."""
 
-        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
-
         # AI Platform service returns 404
         with pytest.raises(exceptions.NotFound):
             aiplatform.ImageDataset(dataset_name="0")
 
-    @pytest.mark.usefixtures("create_text_dataset", "delete_new_dataset")
-    def test_get_new_dataset_and_import(self, dataset_gapic_client, shared_state):
+    def test_get_new_dataset_and_import(self, dataset_gapic_client):
         """Retrieve new, empty dataset and import a text dataset using import().
         Then verify data items were successfully imported."""
 
-        assert shared_state["dataset_name"]
-        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
+        try:
+            text_dataset = aiplatform.TextDataset.create(
+                display_name=f"temp_sdk_integration_test_create_text_dataset_{uuid.uuid4()}",
```

Comment:
Preference to use this shared method for display names: https://github.com/googleapis/python-aiplatform/blob/main/tests/system/aiplatform/e2e_base.py#L50